components/PralineForm/index.js

```diff
@@ -0,0 +1,214 @@
+    setNameField("");
+    setVersionField("");
+    setWeightField("");
+    setCurrentPraline(null);
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <h2>{currentPraline ? "Pralinen bearbeiten" : "Pralinen erstellen"}</h2>
+      <br />
+      <button type="button">Praline bearbeiten</button>
+      <button type="button" disabled={!currentPraline} onClick={handleDelete}>
+        Praline löschen
+      </button>
+      <br />
+      <InputField
+        id="name"
+        label="Name"
+        value={nameField}
+        onChange={(event) => {
+          setNameField(event.target.value);
+        }}
+      />
+      <br />
+      <InputField
+        id="version"
+        label="Version"
+        value={versionField}
+        onChange={(event) => {
+          setVersionField(event.target.value);
+        }}
+      />
+      <br />
+      <InputField
+        id="weight"
+        label="Gewicht"
+        value={weightField}
+        onChange={(event) => {
+          setWeightField(event.target.value);
+        }}
+      />
+      <br />
+      <IngredientList
+        label="Zutaten"
+        id="ingredient"
+        onAddIngredient={handleAddIngredient}
+        ingredients={ingredients}
+        onDeleteIngredient={handleDeleteIngredient}
+      />
+      <br />
+      <IngredientList
+        label="Allergenspuren"
+        id="traces"
+        placeholder="z.B. Schalenfrüchte"
+        onAddIngredient={handleAddAllergyTraces}
+        ingredients={allergyTraces}
+        onDeleteIngredient={handleDeleteAllergyTrace}
+      />
+      <br />
+      Bild hochladen:{" "}
+      <CldUploadButton
+        uploadPreset="lyzzky1u"
+        onUpload={({ info }) => setImageId(info.public_id)}
+      />
+      <br />
+      <button type="button" onClick={cancel}>
+        Zurücksetzen
+      </button>
+      <button type="submit">Speichern / hinzufügen</button>
+      <br />
+      <hr />
+      <p>
+        Hilfsweise Darstellung der Pralinen aus der Datenbank (nur zu
+        Testzwecken):
+      </p>
+      <ul>
+        {data.map((praline) => (
+          <li key={praline._id}>
+            <p>{praline.name}</p>
+
+            {praline.imageId && (
+              <Image
+                width="100"
+                height="100"
+                src={`https://res.cloudinary.com/dtz3vpjks/image/upload/v1691655286/${praline.imageId}.png`}
+                sizes="50vw"
+                alt={praline.name}
+              />
+            )}
+            <button
+              type="button"
+              onClick={() => {
+                setCurrentPraline(praline);
+                setNameField(praline.name);
+                setVersionField(praline.version);
+                setWeightField(praline.weight);
+                setIngredients(praline.ingredients);
+                setAllergyTraces(praline.allergyTraces);
+              }}
+            >
+              bearbeiten
+            </button>
+          </li>
+        ))}
+      </ul>
```

Comment:
When I smell lasagne and I see lasagne, I think that it's actually lasagne that's in the oven. But it's not like that with your form component 😆 

When you create a component and call it `PralineForm` I would expect that the component only contains a `form`. But it contains a list of data as well. Put the list elsewhere, for example on the main page just below the form component.

Think of it like, you're advertising something on your packaging but you're stuffing more into the package than necessary. All I want is lasagne. Give me lasagne!